recipes-core/packagegroups/packagegroup-ni-desirable.bb

```diff
@@ -59,3 +59,8 @@ RDEPENDS:${PN} += "\
 RDEPENDS:${PN}:append:x64 = "\
 	kernel-test-nohz \
 "
+
+# Thunderbolt support packages
+RDEPENDS:${PN} += "\
```

Comment:
Discussions around whether we're going to ship bolt with BSI or put it in feeds are ongoing.
Will update this PR and close this issue when we decide what approach we want to take.

If we decide to ship in feeds, I'd like to move this to core feeds so it is available in offline feeds.
Seeking opinions on this and which packagegroup this should go into.

Reply:
@amstewart any thoughts on which packagegroup this should go into?

Reply:
We don't have a good packagegroup for it, unfortunately. The best place at this time is probably directly into the [core packagefeed](https://github.com/ni/meta-nilrt/blob/nilrt/master/kirkstone/recipes-core/packagegroups/packagefeed-ni-core.bb) (like `dkms`).